### ISSUE TITLE
Using this for an IdP initiated sign in

### ISSUE THREAD

**Issue by amitej1** [apokalipto/devise_saml_authenticatable#155]
How can we use this gem for IdP initiated sign in? 
From my understanding we should call the ACS URL directly which would have the SAMLResponse. But when we tried to do that, it is throwing an error: saml: Auth errors: Invalid SAML Response. Not match the saml-schema-protocol-2.0.xsd

Any pointers?

**Comment by adamstegman** [comment]
Yes, your IdP should send you to the ACS URL along with a SAML response. It looks like the response your IdP is sending doesn't conform to the expected protocol. The ruby-saml gem does [the validation that's failing here](https://github.com/onelogin/ruby-saml/blob/eae81404d8661046aa053d0f4b4fa80f569f9f17/lib/onelogin/ruby-saml/response.rb#L402). It also looks like [the schema is hard-coded](https://github.com/onelogin/ruby-saml/blob/da2fb2b63f87f409143e41523fa591afcb228c78/lib/onelogin/ruby-saml/saml_message.rb#L27), so it's not a matter of choosing a different one. 

I would suggest you inspect the SAML response from your IdP (you could do that by adding a logging statement to [the strategy](https://github.com/apokalipto/devise_saml_authenticatable/blob/2105b89e2117ce6e0761c577ea6a810931295b31/lib/devise_saml_authenticatable/strategy.rb#L9)), and see if you can spot the error, or send it to your IdP to have them figure out why it's invalid.
